include/h2o/socket.h

```diff
@@ -108,6 +108,11 @@ struct st_h2o_socket_t {
      * total bytes written (above the TLS layer)
      */
     size_t bytes_written;
+    /**
+     * flag to track if sock is being traced
```

Comment:
What is the type of the flag? I'd appreciate having that clarified in the doc-comment.

And assuming that it is a boolean, the values being assigned to the value should be either 0 (meaning false) or 1 (meaning true).